src/controllers/StudentController.js

```diff
@@ -0,0 +1,160 @@
+import StudentService from '../services/StudentService.js';
+
+import { z } from 'zod';
```

Comment:
Padronizar importações de libs antes das importações do projeto.

```suggestion
import { z } from 'zod';

import StudentService from '../services/StudentService.js';
```